invoices_test.go

```diff
@@ -846,3 +846,26 @@ func TestInvoices_RefundVoidOpenAmount_Params(t *testing.T) {
 		t.Fatal("expected create open amount refund to return OK")
 	}
 }
+
+func TestInvoices_RecordPayment(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/v2/invoices/1402/transactions", func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != "POST" {
+			t.Fatalf("unexpected method: %s", r.Method)
+		}
```

Comment:
Let's assert the request body matches the expected values so we can confirm the method sends what we expect.